test-e2e/manager-invite.js

```diff
@@ -1,166 +1,181 @@
   )
 
-  const joinerProject = await joiner.getProject(
-    joinerListedProjects[0].projectId
-  )
+  const joinerProject = await joiner.getProject(createdProjectId)
 
-  t.ok(joinerProject, 'can create joiner project instance')
+  t.alike(
+    await joinerProject.$getProjectSettings(),
+    await creatorProject.$getProjectSettings(),
+    'Project settings match'
+  )
 
-  // TODO: Get project settings of joiner and ensure they're similar to creator's project's settings
-  // const joinerProjectSettings = await joinerProject.$getProjectSettings()
-  // t.alike(joinerProjectSettings, { defaultPresets: undefined, name: 'Mapeo' })
+  t.alike(
+    await creatorProject.$member.getMany(),
+    await joinerProject.$member.getMany(),
+    'Project members match'
+  )
 
-  // TODO: Get members of creator project and assert info matches joiner
-  // const creatorProjectMembers = await creatorProject.$member.getMany()
-  // t.is(creatorProjectMembers.length, 1)
-  // t.alike(creatorProjectMembers[0], await joiner.getDeviceInfo())
+  await disconnectPeers([creator, joiner])
 })
 
-test('member invite rejected', async (t) => {
-  t.plan(9)
+test('chain of invites', async (t) => {
+  const managers = await createManagers(4)
+  const [creator, ...joiners] = managers
+  connectPeers(managers)
+  await waitForPeers(managers)
 
-  const deferred = pDefer()
+  const createdProjectId = await creator.createProject({ name: 'Mapeo' })
 
-  const creator = new MapeoManager({
-    rootKey: KeyManager.generateRootKey(),
-    dbFolder: ':memory:',
-    coreStorage: () => new RAM(),
-  })
+  let invitor = creator
+  for (const joiner of joiners) {
+    const invitorProject = await invitor.getProject(createdProjectId)
+    const responsePromise = invitorProject.$member.invite(joiner.deviceId, {
+      roleId: COORDINATOR_ROLE_ID,
+    })
+    const [invite] = await once(joiner.invite, 'invite-received')
+    await joiner.invite.accept(invite.projectId)
+    t.is(
+      await responsePromise,
+      InviteResponse_Decision.ACCEPT,
+      'correct invite response'
+    )
+  }
+
+  /// After invite flow has completed...
+
+  const creatorProject = await creator.getProject(createdProjectId)
+  const expectedProjectSettings = await creatorProject.$getProjectSettings()
+  const expectedMembers = await creatorProject.$member.getMany()
+
+  for (const joiner of joiners) {
+    const joinerProject = await joiner.getProject(createdProjectId)
+
+    t.alike(
+      await joinerProject.$getProjectSettings(),
+      expectedProjectSettings,
+      'Project settings match'
+    )
+
+    const joinerMembers = await joinerProject.$member.getMany()
+    t.alike(
+      joinerMembers.sort(memberSort),
+      expectedMembers.sort(memberSort),
+      'Project members match'
+    )
+  }
+
+  await disconnectPeers(managers)
+})
 
-  await creator.setDeviceInfo({ name: 'Creator' })
+// Needs fix to inviteApi to check capabilities before sending invite
```

Comment:
Mostly commenting so that the relevant issue has a reference to this after it's implemented: https://github.com/digidem/mapeo-core-next/issues/299

```suggestion
// Needs fix to inviteApi to check capabilities before sending invite
```